From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 29 Jan 2020 20:33:50 +0100
Subject: ANDROID: fix build problem in
 include/trace/events/android_fs_template.h

How did this ever work???

Cc: Mohan Srinivasan <srmohan@google.com>
Cc: Amit Pundir <amit.pundir@linaro.org>
Cc: Alistair Strachan <astrachan@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I06279d79877d2e312a458a4d8006d3aaa8e43a60
---
 include/trace/events/android_fs_template.h | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/include/trace/events/android_fs_template.h b/include/trace/events/android_fs_template.h
--- a/include/trace/events/android_fs_template.h
+++ b/include/trace/events/android_fs_template.h
@@ -8,13 +8,13 @@ DECLARE_EVENT_CLASS(android_fs_data_start_template,
 		 pid_t pid, char *pathname, char *command),
 	TP_ARGS(inode, offset, bytes, pid, pathname, command),
 	TP_STRUCT__entry(
-		__string(pathbuf, pathname);
-		__field(loff_t,	offset);
-		__field(int,	bytes);
-		__field(loff_t,	i_size);
-		__string(cmdline, command);
-		__field(pid_t,	pid);
-		__field(ino_t,	ino);
+		__string(pathbuf, pathname)
+		__field(loff_t,	offset)
+		__field(int,	bytes)
+		__field(loff_t,	i_size)
+		__string(cmdline, command)
+		__field(pid_t,	pid)
+		__field(ino_t,	ino)
 	),
 	TP_fast_assign(
 		{
@@ -45,9 +45,9 @@ DECLARE_EVENT_CLASS(android_fs_data_end_template,
 	TP_PROTO(struct inode *inode, loff_t offset, int bytes),
 	TP_ARGS(inode, offset, bytes),
 	TP_STRUCT__entry(
-		__field(ino_t,	ino);
-		__field(loff_t,	offset);
-		__field(int,	bytes);
+		__field(ino_t,	ino)
+		__field(loff_t,	offset)
+		__field(int,	bytes)
 	),
 	TP_fast_assign(
 		{
